students/g_rama/lesson03/list_lab.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+###Series 1####
+
+fruits = ['Apples', 'Pears', 'Oranges', 'Peaches']
+print(fruits)
+
+response_fruit = input("Which fruit do you want to add?")
+if response_fruit is not '':
+    fruits.append(response_fruit)
+
+print(fruits)
+
+try:
+    response_no = int(input("Which Number of Fruit do you want?"))
+    print(fruits[response_no - 1])
+except ValueError:
+    print("expecting an integer value")
+
+response_fruit = input("Which fruit do you want to add at the beginning?")
+if response_fruit is not '':
```

Comment:
`is` is used for object comparison use `var != ''` for value comparison

in this case you can just do `if response_fruit:`